install-habitat/cmd/root.go

```diff
@@ -0,0 +1,123 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"regexp"
+	"runtime"
+
+	"github.com/chef/ci-studio-common/lib"
+
+	"github.com/juju/fslock"
+	"github.com/spf13/cobra"
+)
+
+var (
+	rootCmd = &cobra.Command{
+		Use:   "install-habitat",
+		Short: "Install VERSION of Chef Habitat from CHANNEL.",
+		Run:   maybeInstallHabitat,
+	}
+
+	rootOpts = struct {
+		version string
+		channel string
+		target  string
+	}{}
+
+	globalHabValue = "0.82.0"
```

Comment:
Could we add a comment/note here that this value is modified by an expeditor script?